fuzzer/README.md

```diff
@@ -1,3 +1,8 @@
 ## fuzz_json
 This fuzzer creates a json file directly from bytes.
 `HFUZZ_RUN_ARGS="--dict=json.dict" cargo hfuzz run fuzz_json`
+
+## cairo_compiled_programs_fuzzer
+To run this fuzzer you need to be inside a py_env and be able to run cairo-compile command from the fuzzer folder beforehand.
+
+To run simply use `cargo +nightly fuzz run --fuzz-dir . cairo_compiled_programs_fuzzer`
```

Comment:
I ran `cargo +nightly install fuzz`, the execute this command and got

`[should i run ](error: no such command: `fuzz`)`

Reply:
I would guess it depends on `cargo-fuzz` and it needs to be installed first? In that case, we need to add it to the dependencies and document that need.

Reply:
Also, it would be desirable to document why exactly we need to use nightly here.

Reply:
the command to install the tool to run this fuzzer is `cargo +nightly install cargo-fuzz`, i can add a command in the makefile to install the tool and other to run the fuzzer

Reply:
added the documentation and the makefile commands!